worlds/loonyland/__init__.py

```diff
@@ -0,0 +1,156 @@
+        "monster_dolls": {name for name, data in loony_item_table.items() if data.category == LLItemCat.DOLL},
+        "cheats": {name for name, data in loony_item_table.items() if data.category == LLItemCat.CHEAT},
+        "special_weapons": {
+            name
+            for name, data in loony_item_table.items()
+            if VAR_WBOMBS <= data.id - loonyland_base_id <= VAR_WHOTPANTS
+        },
+        "power": {name for name, data in loony_item_table.items() if "PWR" in data.flags},
+        "power_big": {name for name, data in loony_item_table.items() if "PWR_BIG" in data.flags},
+        "power_max": {name for name, data in loony_item_table.items() if "PWR_MAX" in data.flags},
+    }
+
+    location_name_groups = {
+        "quests": {name for name, data in loonyland_location_table.items() if data.category == LLLocCat.QUEST},
+        "badges": {name for name, data in loonyland_location_table.items() if data.category == LLLocCat.BADGE},
+    }
+
+    def create_junk(self) -> LoonylandItem:
+        return LoonylandItem("A Cool Filler Item", ItemClassification.filler, loonyland_base_id + 3000, self.player)
+
+    def create_item(self, name: str) -> LoonylandItem:
+        if name == "A Cool Filler Item":
+            return self.create_junk()
+        return LoonylandItem(
+            name, loony_item_table[name].modified_classification(self.options), loony_item_table[name].id, self.player
+        )
+
+    def create_items(self) -> None:
+        item_pool: list[LoonylandItem] = []
+        for name, item in loony_item_table.items():
+            if item.id and item.can_create(self.options) and item.in_logic(self.options):
+                for i in range(item.frequency):
+                    new_item = self.create_item(name)
+                    item_pool.append(new_item)
+
+        junk_len = len(self.multiworld.get_unfilled_locations(self.player)) - len(item_pool)
+        if self.options.win_condition == WinCondition.option_evilizer:
+            junk_len = junk_len - 1
+        item_pool += [self.create_junk() for _ in range(junk_len)]
+
+        self.multiworld.itempool += item_pool
+
+    def create_event(self, event: str) -> LoonylandItem:
+        return LoonylandItem(event, ItemClassification.progression, None, self.player)
+
+    def create_regions(self) -> None:
+        # print(self.player_name)
+        # for key in LoonylandOptions.__annotations__:
+        #    print(key, getattr(self.options, key))
+
+        for region_name, region_data in loonyland_region_table.items():
+            if region_data.can_create(self.options):
+                region = Region(region_name, self.player, self.multiworld)
+                self.multiworld.regions.append(region)
+
+        for loc_name, loc_data in loonyland_location_table.items():
+            if not loc_data.can_create(self.options):
+                continue
+            region = self.multiworld.get_region(loc_data.region, self.player)
```

Comment:
```suggestion
            region = self.get_region(loc_data.region)
```
Just a shorter version of it